sroka/api/ga/README.md

```diff
@@ -37,3 +37,46 @@ request = {
 df_ga = ga_request(request, print_sample_size=True, sampling_level='FASTER')
 
 ```
+
+
+### ga_request_all_data(input_dict, start_index, page_size, max_pages, print_sample_size, sampling_level):
```

Comment:
Could you please put this function in quotes so that it's formatted as code (just as ga_request function is above?)